### PR TITLE
add apr snippet file per bug 1446192

### PR DESCRIPTION
**Pull request by peiying2**
@flod, @TheoChevalier 

I want to get a head-start knowing there will be revisions.
1). Firefox Rocket is for id only, so we will have two versions. Consider this the "id" file for now, and I will separate it from the others when we have all the strings cleaned up. (will ask her in the bug)
2). I shorten a lot of notes and some are not necessary. See if they are sufficient and clear.
3). The one with YouTube link, check on your end to see if you have French or Italian subtitles when you play it. Is that what she meant? I don't quite understand her note. 
4).  No character limitation mentioned in the instruction. I will check. The last one is really long.

### CODE REVIEW

File: en-US/snippets/2018/apr2018.lang [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+# Campaign: MDN
+# Link: https://developer.mozilla.org/docs/Games
+;Want to learn how to make games for the web? <a>MDN web docs</a> has you covered.
+Want to learn how to make games for the web? <a>MDN web docs</a> has you covered.
+
+
+# Campaign theme: delight
+# Link: https://youtu.be/FqnWEWYp9DU
+# The video sound track is in English. However, the localized Firefox browser will dictate the language the subtitles are in.
```
Comment (flodolo) [inline en-US/snippets/2018/apr2018.lang]: The video will have subtitles if you manually enable them, and I don't think that's possible for a link. That's only possible if you embed the video in a page.

Honestly, I don't see much of a point in sending people to such video.

File: en-US/snippets/2018/apr2018.lang [hunk 2]
```diff
@@ -0,0 +1,61 @@
+## NOTE: Reference bug 1446192; Deadline: 9 April, 2018
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: cross platform promotions on a variety of topics
+## NOTE: Try to limit character length to 180, preferably under 160.
```
Comment (flodolo) [inline en-US/snippets/2018/apr2018.lang]: Try to limit is never going to work IMO. You should put a MAX_LENGTH on each one, to make they don't go over 180 at least.

**Comment by peiying2** [comment]
Chatted with her on Slack, she has agreed to remove the hamburger snippet, but assures us that the video works in target languages.

File: en-US/snippets/2018/apr2018.lang (hunk 2 above)
Comment (peiying2) [reply en-US/snippets/2018/apr2018.lang]: let me fix this

**Comment by flodolo** [comment]
> Chatted with her on Slack, she has agreed to remove the hamburger snippet, but assures us that the video works in target languages.

Please convince her to use Bugzilla in the future, decisions need to be documented and owned ;-)

**Comment by peiying2** [comment]
Yes, I did, I want it to come from her so it is official.

**Comment by peiying2** [comment]
after you approve it, i will spin off the id version from the rest.

**Comment by flodolo** [comment]
Looks good, thanks. Can you add the files to all languages?

**Comment by peiying2** [comment]
actually committed all languages.

**Pull request merged**